### PR TITLE
Restore header to TSV output

### PR DESCRIPTION
**Pull request by ajnelson-nist**

### CODE REVIEW

**Comment by ajnelson-nist** [comment]
@simsong - The header on the TSV output disappeared in a recent PR from you, and this broke a consumer process I have that relied on the field names being present.  Could you please merge this PR to restore the header?

**Comment by simsong** [comment]
I think that the PR came when the DictWriter was implemented?
Let's update the test suite to check for the header. DO you want to do that?
________________________________

**Comment by simsong** [comment]
I think that the PR came when the DictWriter was implemented? 
Let's update the test suite to check for the header. DO you want to do that?

**Comment by ajnelson-nist** [comment]
I upped the expected row count in the test suite.  I wouldn't mind checking for the header a bit more specifically, but I'm fine with this merging as-is.

File: tests/app_test.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def test_index_tsv():
     with boddle(params={'row_count':'5', 'offset':'0'}):
         res = bottle_app.index_tsf()
         lines = res.split('\n')
-        assert len(lines) == 6
+        assert len(lines) == 7
```
Comment (simsong) [inline tests/app_test.py:46]: Ugh. My `assert len(lines)==6` was based on the idea that there were 5 lines. However, a line terminated by `\n` will result in an empty:
```
>>> "this\n".split('\n')
['this', '']
>>>
```

**Review by simsong — APPROVED** [review]

**Pull request merged**